engine/src/main/java/org/camunda/bpm/engine/impl/db/EnginePersistenceLogger.java

```diff
@@ -800,5 +800,13 @@ public void ignoreFailureDuePreconditionNotMet(DbOperation ignoredOperation, Str
         preconditionMessage,
         failedOperation.toString());
   }
+  
+  public void logTaskAlreadyFetched(String workerId, Throwable cause) {
+	  logWarn("091",
+			  "Execution Null. External task {}  was already fetched by worker thread {} {}", 
+			  workerId, 
+			  cause.getMessage(), 
+			  cause);
+  }
```

Comment:
In the case of optimistic lokcing exception on the task itself, we currently do not log anything. In that sense, I would be ok to also not log anything in this case. If we do want to log though, it should be on level `DEBUG` as this is an expected condition in high concurrency scenarios and doesn't pose a problem that someone must take care of.